server/db/cohorts.js

```diff
@@ -1,4 +1,7 @@
-const connection = require('./')
+const environment = 'development'
```

Comment:
Please utilise the database config already built into `db/index.js`. This is to avoid duplicated code as we will need the same config in each new db module.